src/coord/config.c

```diff
@@ -51,15 +51,27 @@ CONFIG_GETTER(getClusterTimeout) {
 }
 
 CONFIG_SETTER(setGlobalPass) {
+  RedisModule_Log(RSDummyContext, "warning",
+    "OSS_GLOBAL_PASSWORD is deprecated. Use `CONFIG SET search-oss-global-password <password>` instead");
   SearchClusterConfig *realConfig = getOrCreateRealConfig(config);
   int acrc = AC_GetString(ac, &realConfig->globalPass, NULL, 0);
   RETURN_STATUS(acrc);
 }
 
 CONFIG_GETTER(getGlobalPass) {
+  RedisModule_Log(RSDummyContext, "warning",
```

Comment:
Need to make sure this the desired log level

Reply:
```
# notice (moderately verbose, what you want in production probably)
# warning (only very important / critical messages are logged)
```
We can consider notice, but warning seems good so users handle it before being the deprecated configuration is dropped.